==== triton/lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp#30 - triton/lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp ====
# action=edit type=text
--- triton/lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp	2024-03-11 11:42:57.000000000 -0700
+++ triton/lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp	2024-03-15 06:57:27.000000000 -0700
@@ -510,10 +510,8 @@
                                    ConversionPatternRewriter &rewriter,
                                    Type elemTy, MultipleOperandsRange operands,
                                    Location loc) const {
-    auto boolFalse = rewriter.getBoolAttr(false);
-    auto constFalse = rewriter.create<LLVM::ConstantOp>(loc, boolFalse);
     return {rewriter.create<LLVM::AbsOp>(loc, elemTy, operands[0][0],
-                                         /*is_int_min_poison=*/constFalse)};
+                                         /*is_int_min_poison=*/false)};
   }
 };
 
